react-pr2/src/features/Cart/components/CartEmpty.jsx

```diff
@@ -0,0 +1,39 @@
+import { Box, Button } from '@mui/material';
+import { makeStyles } from '@mui/styles';
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import { useTranslation } from 'react-i18next';
+
+const useStyles = makeStyles({
+  root: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    gap: '12px',
+    padding: '40px',
+  },
+});
+
+function CartEmpty(props) {
+  const classes = useStyles();
+  const history = useHistory();
+  const { t } = useTranslation();
+
+  const handleCartClick = () => {
+    history.push('/');
+  };
+
+  return (
+    <Box className={classes.root}>
+      <Box>
+        <img src="https://salt.tikicdn.com/desktop/img/mascot@2x.png" alt="cart" width="190px" />
+      </Box>
+      <Box component="p">{t('There are no products in your cart')}</Box>
+      <Button variant="contained" onClick={handleCartClick}>
```

Comment:
```suggestion
      <Button as='link' href='/' variant="contained" onClick={handleCartClick}>
```
Mui có kiểu này k nhỉ